chaos-monkey-docs/src/main/asciidoc/watcher.adoc

```diff
@@ -14,6 +14,13 @@ With the help of https://docs.spring.io/spring/docs/current/spring-framework-ref
 execution of a _public_ method and will either not execute any action or start one of its assaults. You can customize the behave by
 <<configuration,configuration>>.
 
+Besides the _AOP_ based watchers, Chaos Monkey for Spring Boot also provides watchers for:
+
+- RestTemplate
+- WebClient
+
+The watchers will inject exception and latency based assaults for all _RestTemplate_ and _WebClient_ beans in the Spring Context based on the given assault configuration.
```

Comment:
We might want to add that the the builders are also instrumented and a warning that the clients instantiated and used within  a class (`WebClient.create() and `new RestTemplate`) are not instrumented but can be if the filter is added manually.

Reply:
done